This file is part of MXE.
See index.html for further information.

Fix WinError.h capitalization. See http://svn.boost.org/trac/boost/ticket/7081

diff -ur a/boost/detail/win/basic_types.hpp b/boost/detail/win/basic_types.hpp
--- a/boost/detail/win/basic_types.hpp	2011-03-25 02:20:55.000000000 +0100
+++ b/boost/detail/win/basic_types.hpp	2013-11-13 18:59:00.225614699 +0100
@@ -14,7 +14,7 @@
 #if defined( BOOST_USE_WINDOWS_H )
 # include <windows.h>
 #elif defined( WIN32 ) || defined( _WIN32 ) || defined( __WIN32__ ) ||  defined(__CYGWIN__)
-# include <WinError.h>
+# include <winerror.h>
 // @FIXME Which condition must be tested
 # ifdef UNDER_CE
 #  ifndef WINAPI
